### PR TITLE
Integrate EF7 beta6

### PR DESCRIPTION
**Pull request by rayao**
Resolved comments in the original CR.
VS must have latest NuGet client tool to install EF7 packages.

### CODE REVIEW

**Comment by rayao** [comment]
Forgot to mention that 6 baseline comparison cases still fail, which is expected for EF7 for now.

**Comment by lewischeng-ms** [comment]
@rayao Please add ignore `.vs/` to `.gitignore` as it is generated by VS2015.

File: src/Microsoft.Restier.EntityFramework/DbDomain.cs [hunk 1]
```diff
@@ -20,6 +20,7 @@ namespace Microsoft.Restier.EntityFramework
     /// <summary>
     /// Represents a domain over a DbContext.
     /// </summary>
+    [CLSCompliant(false)]
```
Comment (congysu) [inline src/Microsoft.Restier.EntityFramework/DbDomain.cs]: Why is DbDomain with `[CLSCompliant(false)]`?

File: src/Microsoft.Restier.EntityFramework7/Model/ModelProducer.cs [hunk 2]
```diff
@@ -16,6 +16,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using EdmModel = Microsoft.OData.Edm.Library.EdmModel;
+using Microsoft.Data.Entity.Infrastructure;
```
Comment (congysu) [inline src/Microsoft.Restier.EntityFramework7/Model/ModelProducer.cs]: Sort using

File: src/Microsoft.Restier.EntityFramework/DbDomain.cs (hunk 1 above)
Comment (rayao) [inline src/Microsoft.Restier.EntityFramework/DbDomain.cs]: EF7 DbContext type is not CLS compliant, and that makes a type references it also not compliant. This has been modified to conditionally with EF7.

Comment (congysu) [inline src/Microsoft.Restier.EntityFramework/DbDomain.cs]: Thanks Ray.

**Pull request merged**